packages/client/src/returns/types/getReturnWorkflow.types.ts

```diff
@@ -0,0 +1,14 @@
+import type { Config } from '../..';
+import type { Return } from './return.types';
+
+export type ReturnWorkflow = {
```

Comment:
Could we define this type instead like this?

```js
export type ReturnWorkflowStep = {
  isCompleted: boolean;
  sort: number;
  status: { code: string };
  statusDateUTC?: string;
};

export type ReturnWorkflow = ReturnWorkflowStep[];
```

Let me know if the name `ReturnWorkflowStep` makes sense as that is what I figured those elements inside the array represent but I can be wrong :)